gsf-core/src/main/java/tools/gsf/navigation/Node.java

```diff
@@ -46,4 +46,24 @@
      * @return this node's children, never null.
      */
     List<NODE> getChildren();
+    
+    /**
```

Comment:
I'm still not satisfied. Think of other options.

For example, what if we created a node "kernel" that was already wired up, and it was passed into the "create" method (renamed appropriately) so that the "create" method instead built a better node around the kernel?

The point is, wiring is entirely contained within the site plan implementation, so there should be no need export any aspect of wiring. 

Thoughts?